pages/portfolio.html

```diff
@@ -4,9 +4,154 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
+  <link href="/styles/portfolio.css" rel="stylesheet">
+  <title>Arika's Portfolio</title>
 </head>
 <body>
+  <section class="container">
+
+    <div class="header">
+      <header>
+        <h1>My Personal Portfolio</h1>
+      </header>
+    </div>
+
+    <div class="navigation-links">
+        <nav>
+          <ul>
+            <li>
+              <a href="http://127.0.0.1:5500/pages/about.html">About</a>
+            </li>
+            <li>
+              <a href="http://127.0.0.1:5500/pages/index.html">Home</a>
+            </li>
+          </ul>
+        </nav>
+    </div>
+
+    <div class="main-body">
```

Comment:
You could consider adding a main tag and section tag here to add more structure to your page. If someone was using a screen reader then they would be able to tell that this is the "main" part of the page and that there is a specific "section" to it. and if you needed to later, you could add more sections too

```html
<div class="main-body">
    <main>
        <section>
               <img src="https://media.istockphoto.com/id/1205513619/vector/laptop-with-code-on-screen-hanging-over-icons-programming-app-software-development-web.jpg?s=612x612&w=0&k=20&c=qA1L3ltCqI_uO0jVAgQ85hAd3_BTuVJPwFaYjEDhP9k=" alt="programming photo" width="700">
               <p>
        This personal portfolio will highlight the projects that I have completed so far as
        a developer. <br> Each section will include a list of the skills that were assessed while
        completing the project. <br> Projects with Github repos will have a clickable title.
               </p>
        </main>
    </section>
</div>